packages/orbit-components/src/Box/normalize.js

```diff
@@ -0,0 +1,102 @@
+// @flow
+import { WIDTH_AND_HEIGHT, TOKENS } from "./consts";
+import type { ThemeProps, Theme } from "../defaultTheme";
+
+import type { MediaQueryObject, Elevation } from "./index";
+
+const jsxPropsMap = {
+  wrap: "flex-wrap",
+  shrink: "flex-shrink",
+  grow: "flex-grow",
+  direction: "flex-direction",
+  align: "align-items",
+  justify: "justify-content",
+  textAlign: "text-align",
+  maxWidth: "max-width",
+  maxHeight: "max-height",
+};
+
+const isJsxProp = (key): boolean => Object.keys(jsxPropsMap).includes(key);
+
+const formatCSS = (key, value): string => `${key}: ${value};`;
+
+const firstToUpper = string => string.charAt(0).toUpperCase() + string.slice(1);
+
+const normalizeSpacing = (el, property, theme: Theme): string[] => {
+  if (typeof el === "object") {
+    return Object.entries(el).map(([key, val]: any) => {
+      return formatCSS(`${property}-${key}`, TOKENS(theme)[val]);
+    });
+  }
+
+  if (el !== "none") {
+    return [formatCSS(property, TOKENS(theme)[el])];
+  }
+
+  return [];
+};
+
+const normalizeToken = (token, property, prefix, theme: Theme): string => {
+  if (token === "radiusCircle") return formatCSS(property, theme.orbit.borderRadiusCircle);
```

Comment:
As suggested before, I still would use `circle` name for this option of `borderRadius`. Because we sticked to `radiusCircle` special condition is needed and I don't see any advantage of it 🤔